couchbase/admin/query/v1/query.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+option go_package = "github.com/couchbase/goprotostellar/genproto/admin_query_v1;admin_query_v1";
+option ruby_package = "Couchbase::Protostellar::Generated::Admin::Query::V1";
+option php_namespace = "Couchbase\\Protostellar\\Generated\\Admin\\Query\\V1";
+option java_package = "com.couchbase.client.protostellar.admin.query.v1";
+option java_multiple_files = true;
+
+package couchbase.admin.query.v1;
+
+service QueryAdminService {
```

Comment:
Are we pushing watch indexes to the SDK?

Reply:
Resolving this conversation in favour of the 'Duplicate' comment from Graham.